app/javascript/src/components/Wrapper/index.jsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import PropTypes from "prop-types";
+
+import SideBar from "../Common/SideBar";
+
+const Wrapper = ({ children }) => {
+  return (
+    <div className="w-full h-screen flex flex-row">
+      <SideBar />
+      <div className="h-full w-full">{children}</div>
+    </div>
+  );
+};
+
+Wrapper.propTypes = {
+  children: PropTypes.node,
+};
+
+export default Wrapper;
```

Comment:
We don't need to put it inside `Wrapper/index.jsx`. 
Please move this file to `Common`.